src/components/Characters/GenderFlower.js

```diff
@@ -1,35 +1,54 @@
-      />
-    </svg>
-  );
-}
-
-const GenderFlower = styled(GenderFlowerArtwork)`
+const Wrapper = styled(animated.span)`
   position: absolute;
   z-index: ${zIndexArtworkMainDecor};
   color: ${props => props.theme.primaryColor};
+  transform-origin: '50% 50%';
 `;
 
+function GenderFlower({ isVisible, className }) {
+  const transitions = useTransition(isVisible, null, {
+    from: { opacity: 0, transform: 'scale(0) rotate(0deg)' },
+    enter: { opacity: 1, transform: 'scale(1) rotate(90deg)' },
+    leave: { opacity: 1, transform: 'scale(0) rotate(0deg)' },
+    config: config.wobbly
```

Comment:
It is possible to customise it even further with **mass**, **tension**, **friction** (explained a bit here https://www.react-spring.io/docs/hooks/api). Maybe it will look even better?

Reply:
Animation tweaked a little bit, based on wobbly effect, but now is longer and more bouncy - 8bbf7727b4d8003900df4f66c7f5696e31df566b